packages/ibm-products/src/components/Datagrid/types/index.ts

```diff
@@ -266,6 +278,10 @@ export interface DataGridState<T extends object = any>
   loadMoreThreshold?: number;
   onRowClick?: (row, event) => void;
   onSort?: boolean;
+  customizeColumnsProps?: {
+    onSaveColumnPrefs?: (args) => void;
+    labels?: ColumnLabels;
+  };
```

Comment:
Also looks like we need to add `CustomizeColumnsButton` prop in `DataGridState` as it is not present in type definition.

Reply:
Done